optaplanner-benchmarks/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>kie-benchmarks</artifactId>
+    <groupId>org.kie</groupId>
+    <version>7.8.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.kie</groupId>
+  <artifactId>optaplanner-benchmarks</artifactId>
+  <packaging>pom</packaging>
+  <name>OptaPlanner Benchmarks Parent</name>
+  <description>This is the multi-module project top-level parent.</description>
+
+  <modules>
+    <module>optaplanner-perf-framework</module>
+    <module>optaplanner-perf-benchmark</module>
+  </modules>
+
+  <properties>
+    <version.optaplanner>7.8.0-SNAPSHOT</version.optaplanner>
```

Comment:
This is the same as kie.bom.version. Could be removed.

Reply:
Accidental leftover, thanks